webapi/webapi-twoinone-xwalk-tests/twoinone/twoinone_keyboard-manual.html

```diff
@@ -0,0 +1,71 @@
+are permitted provided that the following conditions are met:
+
+* Redistributions of works must retain the original copyright notice, this list
+  of conditions and the following disclaimer.
+* Redistributions in binary form must reproduce the original copyright notice,
+  this list of conditions and the following disclaimer in the documentation
+  and/or other materials provided with the distribution.
+* Neither the name of Intel Corporation nor the names of its contributors
+  may be used to endorse or promote products derived from this work without
+  specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY INTEL CORPORATION "AS IS"
+AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ARE DISCLAIMED. IN NO EVENT SHALL INTEL CORPORATION BE LIABLE FOR ANY DIRECT,
+INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,
+EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+-->
+
+<html>
+<head>
+    <style>
+        * {
+            font-family: Calibri, sans-serif
+        }
+        input, p {
+            font-size: larger
+        }
+    </style>
+</head>
+<body>
+    <h3>Test steps:</h3>
+    <textarea rows="5" cols="90">
+    1. Insert a keyboard to the device
+    Expect: Status of "twoinone" extension is ready.
+    Expect: Status of keyboard watcher shows the keyboard is "hardware keyboard attached".
+    2. Unplug the keyboard from the device
+    Expect: Status of "twoinone" extension is ready.
+    Expect: Status of keyboard watcher shows the keyboard is "no keyboard".
+    </textarea>
+    <h1>Keyboard Monitor Extension</h1>
+    <p>
+        Status of &quot;twoinone&quot; extension: <input style="border:none" id="status" type="text" value="" readonly>
+        <script>
+            document.getElementById('status').value = twoinone ? 'ready' : 'loading failed';
+        </script>
+    </p>
+    <p>
+        Status of keyboard watcher: <input style="border:none; font-weight: bold; color: red" id="keyboard" type="text" value="loading ..." size="30" readonly>
+        <script>
+            twoinone.monitorKeyboard(function (haveKeyboard) {
+                document.getElementById('keyboard').value = haveKeyboard ?
+                                                            "hardware keyboard attached" :
+                                                            "no keyboard";
+            });
+        </script>
+    </p>
+    <p>
+        <input type="button" onclick="alert(twoinone.haveKeyboard())" value="Query Keyboard Availability">
```

Comment:
alert is not a good way to present test result since it may be blocked in some UAs.